### PR TITLE
feat/cta

### PR DESCRIPTION
**Pull request by JonathaGomes**
## What is the reffered issue?

Issue: #21

## What does this do?

This component is a snippet of [landing page](https://www.figma.com/file/pSuBkbjhzZdq0nvHhDFbk7/crypter?node-id=1376%3A36337)

I had a problem with the font in the storybook, but I think I solved it by adding @import from the fonts to globalStyles.ts

## Review Checklist

- [ ] Added and/or modified tests for affected areas
- [x] Ready to review
- [x] Ready to merge

### CODE REVIEW

**Review by joaopaulomoraes — CHANGES_REQUESTED** [review]

File: packages/frontend/src/styles/globalStyles.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { createGlobalStyle } from 'styled-components'
 
 export const GlobalStyles = createGlobalStyle`
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@700&family=Poppins:wght@400;500;600;700&display=swap');
```
[the remark this answers is not in the document]
Comment (JonathaGomes) [reply packages/frontend/src/styles/globalStyles.ts:4]: Yes, but the storybook cannot get a font without this @import

File: packages/frontend/src/components/Cta/styles.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  margin-top: 2.5rem;
+  gap: 1rem;
+  @media (max-width: ${({ theme }) => theme.breakpoints.sm}) {
+    flex-direction: column;
+  }
+`
+
+export const ButtonEarn = styled.button`
+  background: ${({ theme }) => theme.colors.primary.blue};
+  color: #fff;
```
[the remark this answers is not in the document]
Comment (JonathaGomes) [reply packages/frontend/src/components/Cta/styles.ts:127]: Thank You

File: packages/frontend/src/styles/globalStyles.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (joaopaulomoraes) [reply packages/frontend/src/styles/globalStyles.ts:4]: This looks like a storybook issue, I think we can break it down into a new issue.

**Review by joaopaulomoraes — CHANGES_REQUESTED** [review]

**Comment by joaopaulomoraes** [comment]
Thanks @JonathaGomes, nice job!

**Review by CarmichaelF — APPROVED** [review]
Looks like a pretty solid component, thanks for your work here. 👍

**Pull request merged**